spec/models/user_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+
+describe User do
+  describe "validating user information" do
+	  it "has a valid factory" do
+	    FactoryGirl.create(:user).should be_valid
```

Comment:
also since you are testing build in validators, I do not see a point of testing them

Reply:
mostly I used it for learning purposes, my model is almost empty, got you :+1: